src/com/vivian/java/leetcode/Q434_NumberofSegmentsinaString.java

```diff
@@ -11,5 +11,30 @@
  * Input: "Hello, my name is John" Output: 5
  */
 public class Q434_NumberofSegmentsinaString {
-
+    public int numberOfSegmentSinaString(String str) {
+        int count = 0;
+        boolean flag=false;
```

Comment:
please comment what's the meaning of flag, what's the use of it, and come up with a meaningful and better naming.